docs/source/usage.rst

```diff
@@ -3,23 +3,44 @@
 pymesync - Communicate with a TimeSync API
 ==========================================
 
+.. contents::
+
 This module provides an interface to communicate with an implementation of the
 `OSU Open Source Lab`_'s `TimeSync`_ API. An example implementation in Node.js
```

Comment:
``An implementation of the TimeSync API, built in Node.js, can be found `on Github`_.``